js/sponsor.js

```diff
@@ -0,0 +1,24 @@
+$(document).ready(function(){
+
+    $.get('./data/sponsors.json',function(response){
+        constructSponsorBlock(response);
+    });
+});
+
+constructSponsorBlock = function(response){
+        var text = "";
+        
+        $.each(response["sponsors"],function(key,value){
+            var title=  "<div class='row text-center'><h3>" + key +" Sponsors</h3>";
+            $.each(value,function(index,item) {
+                var str = "<div class='col-md-4'><img src='"+ response["border"] +"' class='border2 border2a' alt='"+item["alt-text"]+"'>\
+                <img src='"+ item.img +"' alt='"+item["alt-text"]+"' class='img-responsive sponsora center-block'></div>";
+                title +=str;
+                str = "";
+            });
+            text += title + "</div><br /><br /><br /><br />";
+            title = "";
+        });
+            
+        $("#sponsors-content").append(text);
+}
```

Comment:
Please add a newline here.

Reply:
Adding new line meaning an clrf after the last closing parenthesis?